### PR TITLE
Clarify use of enable_uart and core_freq options

### PR DESCRIPTION
**Pull request by andrum99**
Recently a forum user got confused by this part of the docs as they are currently written. This change rewords it so it is a bit less confusing.

### CODE REVIEW

File: configuration/uart.md [hunk 1]
```diff
@@ -2,17 +2,15 @@
...
+By default, on Raspberry Pis equipped with the wireless/Bluetooth module (Raspberry Pi 3 and Raspberry Pi Zero W), the PL011 UART is connected to the BT module, while the mini UART is used as the primary UART and will have a Linux console on it. On all other models the PL011 is used as the primary UART. 
 
-In Linux device terms, by default, /dev/ttyS0 refers to the mini UART, and /dev/ttyAMA0 refers to the PL011. The primary UART is that assigned to the Linux console, which depends on the Raspberry Pi model as described above, and can be accessed via /dev/serial0.
+In Linux device terms, by default, /dev/ttyS0 refers to the mini UART, and /dev/ttyAMA0 refers to the PL011. The primary UART is that assigned to the Linux console, which depends on the Raspberry Pi model as described above. There are also symlinks /dev/serial0 which always refers to the primary UART (if enabled), and dev/serial1 which similarly always refers to the secondary UART (if enabled).
```
Comment (pelwell) [inline configuration/uart.md]: This would benefit from a '-' after symlinks and a '/' on 'dev/serial1'.

File: configuration/uart.md [hunk 2]
```diff
@@ -2,17 +2,15 @@
...
 
 ## Mini UART and CPU core frequency
 
-The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. Also, when the Linux console uses the mini UART (Raspberry Pi 3, Raspberry Pi Zero W), as a consequence of the UART being disabled, the console is also disabled.
+The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. By default, if the mini UART is selected for use as the primary uart it will be disabled. To enable it, add enable_uart=1 to config.txt. This will also fix the core frequency to 250MHz (unless force_turbo is set, when it will be fixed to 400MHz). When the mini UART is not the primary uart, for example you are using it to connect to the Bluetooth controller, you must add core_freq=250 to config.txt otherwise the mini UART will not work.
```
Comment (pelwell) [inline configuration/uart.md]: I would rather say "the VPU turbo frequency" than "400MHz". Also "for example you are using" -> "for example if you are using".

**Comment by JamesH65** [comment]
@andrum99 Woudl it be possible for you to make the canges as suggested by @pelwell, then I can push to the copy editors.

File: configuration/uart.md (hunk 1 above)
Comment (andrum99) [reply configuration/uart.md]: done

File: configuration/uart.md (hunk 2 above)
Comment (andrum99) [reply configuration/uart.md]: done

**Comment by andrum99** [comment]
Changes made as requested - sorry about the delay.

**Comment by pelwell** [comment]
Thanks - LGTM.

**Pull request merged**